orcestra_book/plans/HALO-20240810a.md

```diff
@@ -0,0 +1,60 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+platform: HALO
+flight_id: HALO-20240810a
+takeoff: 2024-01-01 00:00
+landing: 2020-01-01 01:00
+crew:
+  - name: John Doe
+    job: PI
+  - name: Emily Taylor
+    job: WALES
+  - name: Liam Brooks
+    job: HAMP
+  - name: Ava Reyes
+    job: Dropsondes
+  - name: Ethan Wallace
+    job: Smart/VELOX
+  - name: Ruby Patel
+    job: SpecMACS
+  - name: Logan Nguyen
+    job: Scientist
+  - name: Julia Santos
+    job: Ground contact
+orphan: true
+---
+
+# Flight plan - {front}`flight_id`
+
+{badges}`_`
+
+## Crew
+
+The flight is planned to take off at {front}`takeoff`.
```

Comment:
It doesn't seem like the crew listing would show up here. I'd suggest to either provide a way to include the crew listing here, or to write the crew listing within the markdown part instead of in the front matter.

Reply:
That's also true. In general, the appearance of the flight plans was severely under-prioritized compared to the implementation of the new infrastructure. But I will at least add an ad-hoc table for the first draft